src/update_engine/update_attempter.cc

```diff
@@ -125,7 +125,15 @@ UpdateAttempter::UpdateAttempter(SystemState* system_state,
 
 void UpdateAttempter::Update(bool interactive) {
   fake_update_success_ = false;
+
   if (status_ == UPDATE_STATUS_UPDATED_NEED_REBOOT) {
+    // we init the request_params to generate the correct payload
+    // and provide the correct `update_url`
```

Comment:
```suggestion
    // and provide the correct `update_url`
    http_response_code_ = 0;
```
The HTTP response code should also be reset as the function does. It is used in `ComputeNextIntervalAndFuzz` to start the exponential backoff.